README.md

```diff
@@ -1,13 +1,57 @@
-[![Build Status](https://travis-ci.org/dart-lang/source_gen.svg?branch=master)](https://travis-ci.org/dart-lang/source_gen)
+# source_gen
 
-`source_gen` provides:
+<p align="center">
```

Comment:
Any idea what this looks like on the pub site or other areas? Not sure if its worth this potential headache just to get them centered :).